qutebrowser/config/configdata.yml

```diff
@@ -3194,3 +3194,20 @@ bindings.commands:
 
     * register: Entered when qutebrowser is waiting for a register name/key for
       commands like `:set-mark`.
+
+## logging
+
+logging.level.ram:
+  default: debug
+  type:
+    name: LogLevel
```

Comment:
FYI, you can do `type: LogLevel` directly if you don't want to customize anything about the type. I'll fix this up while merging,t hough.